rust/operator-binary/src/druid_connection_controller.rs

```diff
@@ -22,13 +22,16 @@ use stackable_operator::{
     status::condition::{ClusterConditionStatus, ClusterConditionType},
     time::Duration,
 };
-use stackable_superset_crd::{
-    druidconnection::{DruidConnection, DruidConnectionStatus, DruidConnectionStatusCondition},
-    SupersetCluster, PYTHONPATH, SUPERSET_CONFIG_FILENAME,
-};
 use strum::{EnumDiscriminants, IntoStaticStr};
 
 use crate::{
+    crd::{
+        druidconnection::v1alpha1::{
+            DruidConnection, DruidConnectionStatus, DruidConnectionStatusCondition,
+        },
+        v1alpha1::SupersetCluster,
+        PYTHONPATH, SUPERSET_CONFIG_FILENAME,
+    },
```

Comment:
suggestion: Import the version modules to clearly indicate the version used in downstream code.

```suggestion
    crd::{
        druidconnection, v1alpha1,
        PYTHONPATH, SUPERSET_CONFIG_FILENAME,
    },
```

Reply:
https://github.com/stackabletech/superset-operator/pull/597/commits/21082f3297d85b287bab3468a056b1277b3dfbb8